content/contact.hbs

```diff
@@ -15,9 +15,15 @@
 	<header class="section">
 		<div class="container">
 			<h1>Join in <span class="semester-text">Spring 2021</span> or later</h1>
-			<p class="has-text-centered">To enroll for a future semester, fill out our interest form and we'll contact you with more details including when and how to apply.</p>
+			<p class="has-text-centered">If you are a college or highschool student, to enroll for a future semester, fill out our interest form and we'll contact you with more details including when and how to apply.</p>
 			<p class="has-text-centered"><a class="button is-large future-interest" href="https://docs.google.com/forms/d/e/1FAIpQLSfkvC-q9kdtHttOnJVYtJ4vA26Jpkqbdz1m1LS0h-perXGnhQ/viewform?usp=sf_link">Future Cadet Interest Form</a></p>
 		</div>
+		<div class="container">
+			<h1>Register for the AFOQT</h1>
+			<p class="has-text-centered">If you would like to take the AFOQT here at Det. 165, scan the QR code or click the button below and fill out the poll</p>
+			<img src="img/afoqt.jpg" width="200" height="40" class="center">

```

Comment:
This QR code takes up a lot of unneeded space. Are people really going to scan this instead of clicking the link?

Reply:
It was specifically requested to put the QR, I just added the link just in case